src/html/partials/concept.html

```diff
@@ -1,4 +1,73 @@
 <section class="concept-section">
-  <h2 class="sm-title">The Concept</h2>
-  <p class="bg-intro">Meet Florence</p>
+
+  <div class="concept-section__intro">
+    <h2 class="sm-title">The Concept</h2>
+    <p class="bg-intro">Meet Florence</p>
+
+    <p>Our explorations led to the concept of Florence — a personal companion that would help patients understand and manage their medication regimen. A chatbot at heart, patients would converse with her through text and picture messages.</p>
+      
+    <p>If developed, a simple, familiar interface would hide her powerful intelligence. Behind the scenes, machine learning and behavioral modeling would enable her to offer a tailored, human experience based on each patient’s needs.</p>
+  </div>
+
+  <ul class="concept-section__list">
+    <li class="concept-section__item">
+      <div class="concept-section__item__card">
+        <h3>She Gets To Know You.</h3>
+        <p>Florence strikes up a conversation, learning details about your life, health, goals, and preferences in an engaging, human way.</p>
+        <p>This conversation unfolds over time, making this feel less like twenty questions and more like the start of a relationship.</p>
+      </div>
+      <img src="../../images/sms/Text_SMS_1.png" alt="Hi! I'm Florence. What's your name?">
```

Comment:
For anyone else looking at this PR, we're going with images for the chat bubble screens as a first pass, and having markup based sequenced/animated chat bubbles be a nice-to-have, if there's time or priority later.